tools/probe/probe_vop_contexts.py

```
@@ -0,0 +1,180 @@
+        return self
+
+    def __exit__(self, exc_type, exc, tb):
+        self.cleanup()
+        return False
+
+    def cleanup(self):
+        for root in self.roots:
+            try:
+                root.destroy()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)